composer/cli/launcher.py

```diff
@@ -90,7 +94,7 @@ def parse_args():
 def launch_processes(nproc: int, world_size: int, base_rank: int, master_addr: str, master_port: Optional[int],
                      module_mode: bool, run_directory: Optional[str], training_script: str,
                      training_script_args: List[Any]) -> Set[subprocess.Popen]:
-    print(f"Starting DDP on local node for global_rank({base_rank}-{base_rank+nproc-1})")
+    log.info("Starting DDP on local node for global_rank(%s-%s)", base_rank, base_rank + nproc - 1)
```

Comment:
Is there a reason to prefer C-style string formatting over f-strings?

Reply:
also agree here that I think fstrings are much more understandable and readable

Reply:
I agree, but log calls should use %s because they're lazily evaluated. Granted it doesn't make a big difference, but it is a pylint rule (that's next on my agenda to add pylint to the repo...)